build.psm1

```diff
@@ -1928,12 +1930,13 @@ function Start-TypeGen
     Find-Dotnet
 
     $GetDependenciesTargetPath = "$PSScriptRoot/src/Microsoft.PowerShell.SDK/obj/Microsoft.PowerShell.SDK.csproj.TypeCatalog.targets"
+#            <_RefAssemblyPath Include="%(_ReferencesFromRAR)%3B" Condition=" '%(_ReferencesFromRAR.Type)' == 'assembly' And '%(_ReferencesFromRAR.PackageName)' != 'Microsoft.Management.Infrastructure' " />
     $GetDependenciesTargetValue = @'
 <Project>
     <Target Name="_GetDependencies"
             DependsOnTargets="ResolveAssemblyReferencesDesignTime">
         <ItemGroup>
-            <_RefAssemblyPath Include="%(_ReferencesFromRAR.ResolvedPath)%3B" Condition=" '%(_ReferencesFromRAR.Type)' == 'assembly' And '%(_ReferencesFromRAR.PackageName)' != 'Microsoft.Management.Infrastructure' " />
+            <_RefAssemblyPath Include="%(_ReferencesFromRAR.HintPath)%3B"  Condition=" '%(_ReferencesFromRAR.NuGetPackageId)' != 'Microsoft.Management.Infrastructure' "/>
```

Comment:
@iSazonov for my own education, how did you figure out this change?

Reply:
1. Search "_ReferencesFromRAR" in "C:\Program Files\dotnet\sdk\2.1.300-preview2-008530\" for file filter '*.t*' (targets and tasks)
No useful results.
Star monitoring :-)
2. Run `dotnet msbuild /fileLogger /noconsolelogger /verbosity:diag`
3. Search "_ReferencesFromRAR" in `msbuild1.log`
Look assigned properties

Reply:
You don't have to build powershell to find out this. Use a simple project that targets netcoreapp2.1, then run this:
```
dotnet msbuild .\temp.csproj /t:ResolveAssemblyReferencesDesignTime /fileLogger /noconsolelogger /v:diag
```
Then you can inspect the properties of `_ReferencesFromRAR` in `msbuild.log`. I think we should add comment here to describe this process.